src/styles.tsx

```diff
@@ -6,4 +6,11 @@ const GreyTitle = styled.h1`
   font-family: "Noto Sans";
 `;
 
-export { GreyTitle };
+const BiggerH1 = styled.h1`
+  font-size: 14px;
+  font-family: Rubik, "Noto Sans", -apple-system, BlinkMacSystemFont, "Segoe UI",
```

Comment:
use font from `theme`.

Reply:
It's a styled component, I cant use the theme hook because it needs to be inside a component, and styled components cant be inside react components
![image](https://user-images.githubusercontent.com/54243490/206219622-16385558-f98c-412c-a786-8e42d182d95f.png)

Reply:
You can use `theme` inside react component. Here example

<img width="545" alt="Screenshot 2022-12-07 at 17 31 14" src="https://user-images.githubusercontent.com/11043120/206221503-adc16bbc-9551-4bfd-b669-26a0e851a4a5.png">

Reply:
so take the component H1 and style it